packages/opds-web-client/src/app.tsx

```diff
@@ -66,6 +66,15 @@ class OPDSWebClient {
 
   render() {
     let RouteHandler = this.RouteHandler;
+
+    // `react-axe` should only run in development and testing mode.
+    // Running this is resource intensive and should only be used to test
+    // for accessibility and not while active development.
```

Comment:
Same comment as in registry_admin PR: "while actively developing" or "during active development".